app-config.yaml

```diff
@@ -122,6 +122,12 @@ kafka:
         - localhost:9092
 
 integrations:
+  googleGcs:
+    clientEmail:
+      $env: GCS_CLIENT_EMAIL
```

Comment:
Tbh I don't think we want these in example app config, as you'd want to require these. Perhaps have them here but comment them out.

Could also just inline the email right? Would assume it's not a very secret thing so no need to encourage use of an env var?